src/Subscriber/AutoUpdateTitleWithLabelSubscriber.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace App\Subscriber;
+
+use App\Event\GitHubEvent;
+use App\GitHubEvents;
+use App\Issues\GitHub\CachedLabelsApi;
+use App\Service\LabelNameExtractor;
+use Github\Api\PullRequest;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+/**
+ * When a label changed, then update PR title.
+ *
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ */
+class AutoUpdateTitleWithLabelSubscriber implements EventSubscriberInterface
+{
+    private $labelsApi;
+
+    private $labelExtractor;
+    private $pullRequestApi;
+
+    public function __construct(CachedLabelsApi $labelsApi, LabelNameExtractor $labelExtractor, PullRequest $pullRequestApi)
+    {
+        $this->labelsApi = $labelsApi;
+        $this->labelExtractor = $labelExtractor;
+        $this->pullRequestApi = $pullRequestApi;
+    }
+
+    public function onPullRequest(GitHubEvent $event)
+    {
+        $data = $event->getData();
+        if ('labeled' !== $action = $data['action']) {
+            return;
+        }
+        if (!isset($data['pull_request'])) {
+            // Only update PullRequests
+            return;
+        }
+
+        $originalTitle = $prTitle = $data['pull_request']['title'];
+        $validLabels = [];
+        foreach ($data['pull_request']['labels'] as $label) {
+            if ('dddddd' === $label['color']) {
+                $validLabels[] = $label['name'];
+                // Remove label name from title
+                $prTitle = str_replace('['.$label['name'].']', '', $prTitle);
```

Comment:
If the title is `[fwb] Foo`, then this line will check for `[FrameworkBundle]`, that will not be found and the final title will be: 

`[FrameworkBundle] [fwb] Foo`

Maybe we should remove all `[.*]` in the beginning of the title?

Reply:
This is fixed and I also added a test for it.